js/blocks/controls/email.js

```diff
@@ -1,31 +1,46 @@
+import classNames from 'classnames';
+
 const { TextControl } = wp.components;
 
 const BlockLabEmailControl = ( props, field, block ) => {
 	const { setAttributes } = props;
 	const attr = { ...props.attributes };
+
+	/**
+	 * Sets the Error Class for the Text Control.
+	 *
+	 * @param {string} element The HTML element.
+	 *
+	 * @return {void}
+	 */
+	const setErrorClass = ( element, valid ) => {
```

Comment:
Great call. Let's move this to it's own component and import it!

Reply:
@lukecarbis I want to hold off on this, as I have a different idea of abstracting the entire TextControl into a wrapper component.